### PR TITLE
MOSIP-35416 fix automation failure

### PR DESCRIPTION
**Pull request by anup-nehe**
MOSIP-35416 fix automation failure

### CODE REVIEW

File: api-test/src/main/resources/pms/DeactivateDevice/DeactivateDevice.yml [hunk 1]
```diff
@@ -7,6 +7,7 @@ DeactivateDevice:
       inputTemplate: pms/DeactivateDevice/DeactivateDevice
       outputTemplate: pms/DeactivateDevice/DeactivateDeviceResult
       input: '{
+      "responseTime": "$TIMESTAMP$",
```
Comment (mohanachandran-s) [inline api-test/src/main/resources/pms/DeactivateDevice/DeactivateDevice.yml:10]: Can please adjust the indentation here with respect to line 11

File: api-test/src/main/resources/pms/DeactivateSBIWithAssociatedDevices/DeactivateSBIWithAssociatedDevices.yml [hunk 2]
```diff
@@ -7,6 +7,7 @@ DeactivateSBIWithAssociatedDevices:
       inputTemplate: pms/DeactivateSBIWithAssociatedDevices/DeactivateSBIWithAssociatedDevices
       outputTemplate: pms/DeactivateSBIWithAssociatedDevices/DeactivateSBIWithAssociatedDevicesResult
       input: '{
+      "requesttime": "$TIMESTAMP$",
```
Comment (mohanachandran-s) [inline api-test/src/main/resources/pms/DeactivateSBIWithAssociatedDevices/DeactivateSBIWithAssociatedDevices.yml:10]: Can please adjust the indentation here with respect to line 11

File: api-test/src/main/resources/pms/FilterAPIs/DeviceTypeFilter/DeviceTypeFilter.hbs [hunk 3]
```diff
@@ -12,6 +12,7 @@
       {{#unless @last}},{{/unless}}
       	{{/each}}
     ],
+    "optionalFilters": [],
```
Comment (mohanachandran-s) [inline api-test/src/main/resources/pms/FilterAPIs/DeviceTypeFilter/DeviceTypeFilter.hbs:15]: Can please adjust the indentation here on line 15 & 16

File: api-test/src/main/resources/pms/GetOriginalPartnerCertificates/GetOriginalPartnerCertificates.hbs [hunk 4]
```diff
@@ -1,3 +1,6 @@
 {
+  "id": "mosip.pms.original.partner.certificate.get",
+  "version": "1.0",
+  "requesttime": "{{requesttime}}",
 "partnerId": "{{partnerId}}"
```
Comment (mohanachandran-s) [inline api-test/src/main/resources/pms/GetOriginalPartnerCertificates/GetOriginalPartnerCertificates.hbs:5]: Can please adjust the indentation here

File: api-test/src/main/resources/pms/InactiveMappingDeviceToSbi/InactiveMappingDeviceToSbi.hbs [hunk 5]
```diff
@@ -1,4 +1,8 @@
 {
+"id": "mosip.pms.add.inactive.mapping.device.to.sbi.id.post",
```
Comment (mohanachandran-s) [inline api-test/src/main/resources/pms/InactiveMappingDeviceToSbi/InactiveMappingDeviceToSbi.hbs:2]: Can please adjust the indentation here with respect to line 3

File: api-test/src/main/resources/pms/InactiveMappingDeviceToSbi/InactiveMappingDeviceToSbi.yml [hunk 6]
```diff
@@ -6,6 +6,7 @@ InactiveMappingDeviceToSbi:
       inputTemplate: pms/InactiveMappingDeviceToSbi/InactiveMappingDeviceToSbi
       outputTemplate: pms/InactiveMappingDeviceToSbi/InactiveMappingDeviceToSbiResult
       input: '{
+      "requestTime": "$TIMESTAMP$",
```
Comment (mohanachandran-s) [inline api-test/src/main/resources/pms/InactiveMappingDeviceToSbi/InactiveMappingDeviceToSbi.yml:9]: Can please adjust the indentation here with respect to line 10

**Review by mohanachandran-s — CHANGES_REQUESTED** [review]

**Review by mohanachandran-s — APPROVED** [review]

**Review by mayuradesh — APPROVED** [review]

**Pull request merged**